synapse/storage/databases/main/events_worker.py

```diff
@@ -364,6 +364,10 @@ def get_chain_id_txn(txn: Cursor) -> int:
             replaces_index="event_txn_id_device_id_txn_id",
         )
 
+        # Flag to track when the sliding sync background jobs have
+        # finished (so we don't have to keep querying it every time)
+        self._has_finished_sliding_sync_background_jobs = False
```

Comment:
Random thing we could do. In my editor at-least (VSCode), I get the description in the type hint this way.

```suggestion
        self._has_finished_sliding_sync_background_jobs = False
        """
        Flag to track when the sliding sync background jobs have
        finished (so we don't have to keep querying it every time)
        """
```